src/lex.cpp

```diff
@@ -842,11 +842,12 @@ void lexer::parse_number() {
 
 char8* lexer::parse_decimal_digits_and_underscores(char8* input) noexcept {
   bool has_trailing_underscore = false;
+  const char* garbage_begin = nullptr;
```

Comment:
Blocking: This variable's type should be `const char8*`, not `const char*`. It might work on your machine because your compiler doesn't support `char8_t` (therefore `char8` is the same as `char`), but on CI and on my machine, `char8` and `char` are different types. This leads to a build failure: https://travis-ci.org/github/strager/quick-lint-js/builds/734270869

Reply:
My bad... I forgot to check CI result and only thought that if it is correct on my machine it is universally correct... I will change this habit now.
This should now be corrected!